.github/actions/unit-tests/prepare.sh

```diff
@@ -45,4 +45,6 @@ install_clang
 
 export PATH="/usr/local/clang/bin:$PATH"
 
-go install github.com/mfridman/tparse@28967170dce4f9f13de77ec857f7aed4c4294a5f # v0.12.3 (main) with -progress
+# renovate: datasource=github-release-attachments depName=mfridman/tparse
```

Comment:
I think you'll need to add this file to the `includePaths` and the respective `regexManagers.fileMatch` sections in `.github/renovate.json5` in order for renovate to be able to update this.

Reply:
ah thanks, I always forgot about the above :+1: